### PR TITLE
Update go-control-plane

### PR DESCRIPTION
**Pull request by kyessenov**
Vendor PR https://github.com/istio/vendor-istio/pull/22
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

**Comment by kyessenov** [comment]
circle can't find vendor SHA even though it's in upstream...

**Comment by kyessenov** [comment]
Wonderful, api-server flaked again. cc @costinm @ldemailly

**Comment by ldemailly** [comment]
update vendor to head sha

**Comment by kyessenov** [comment]
Re-approve please.

**Comment by ldemailly** [comment]
can you pull one more time, circle tests are supposed to be fixed now

File: Makefile [hunk 1]
```diff
@@ -643,6 +643,7 @@ show.goenv: ; $(info $(H) go environment...)
 	$(Q) $(GO) version
 	$(Q) $(GO) env
 
+# tickle
```
Comment (ldemailly) [inline Makefile]: remove

Comment (kyessenov) [reply Makefile]: Will do in next PR.

**Comment by ldemailly** [comment]
also please paste the output of:
```
make depend.status
```

**Comment by kyessenov** [comment]
```
make depend.status
No error means your Gopkg.* are in sync and ok with vendor/
dep status -dot > /home/kuat/go/out/linux_amd64/release/dep.dot
cp Gopkg.* vendor/
````

**Pull request merged**